src/lib.rs

```diff
@@ -34,10 +88,15 @@ mod functions {
         fn to_tsquery_with_search_config(config: Regconfig, querytext: Text) -> TsQuery;
     }
     sql_function!(fn to_tsvector(x: Text) -> TsVector);
+    sql_function!(fn nullableto_tsvector(x: Nullable<Text>) -> TsVector);
     sql_function! {
         #[sql_name = "to_tsvector"]
         fn to_tsvector_with_search_config(config: Regconfig, document_content: Text) -> TsVector;
     }
+    sql_function! {
+        #[sql_name = "to_tsvector"]
+        fn nullableto_tsvector_with_search_config(config: Regconfig, document_content: Nullable<Text>) -> TsVector;
+    }
```

Comment:
Instead of having seperate functions for this I would like to handle this similar to how diesel handles other [text functions](https://github.com/diesel-rs/diesel/blob/master/diesel/src/expression_methods/text_expression_methods.rs#L146).
For this case this means that we need to make the functions generic and constrain the text type to some local equivalent of `TextOrNullableText`.